aip/general/0129.md

```diff
@@ -0,0 +1,150 @@
+---
+id: 129
+state: draft
+created: 2023-06-21
+placement:
+  category: resource-design
+  order: 65
+---
+
+# Server-Modified Values and Defaults
+
+Server-modified and default values often result in a poor user experience for
+[state-driven clients](state-driven-clients). These clients are often unable to
+tell when their desired state matches the current state for these fields, as the
+rules by which a server may modify and return values are complex, not public,
+and not repeatable. This AIP aims to clarify what behaviors can be expected from
+services and describing how they can be repeated on the client-side.
+
+## Guidance
+
+### Single Owner Fields
+
+Fields should have a single owner, whether that is the client or the server.
+Server owned fields should be indicated with the `OUTPUT_ONLY` field_behavior.
+All other types of fields should be considered to be owned by the client. The
```

Comment:
```suggestion
All other types of fields **should** be considered to be owned by the client. The
```
I'm guessing this is guidance. If not, maybe change the word to `can` as in `can be considered...`